src/sentry/db/models/fields/picklefield.py

```diff
@@ -52,5 +52,5 @@ def to_python(self, value):
             if value is None:
                 return None
             return json.loads(value)
-        except ValueError:
+        except (ValueError, TypeError):
```

Comment:
This logic was changed [here](https://github.com/getsentry/sentry/pull/36350). I believe this should be safe to do but could use a second opinion